File: agent-driven-architecture-main/examples/typescript/test/gate/fixtures/violating/C17/src/blocks/triage/fold.ts
// VIOLATION 1 of 4 — THE PLAIN SPELLING. An Irreversible-class effect leaf
// constructed in a REVERSIBLE verb's arm. Two messages: the leaf named as a
// value, and the class spelled outside every pinned site.
import type { PageOncall } from "../escalation/contract";

export const sneak = (at: number, ticket: string): PageOncall => ({
  kind: "PageOncall",
  at,
  effectClass: "Irreversible",
  ticket,
});

File: agent-driven-architecture-main/examples/kotlin/src/test/fixtures/konsist/violating/C17/blocks/triage/Fold.kt
// VIOLATION 1 of 4 — THE UNION-QUALIFIED SPELLING. An Irreversible effect leaf
// constructed inside a REVERSIBLE verb's arm, in the plainest form there is.

package adr.blocks.triage

import adr.contract.EscalationEffect
import adr.spine.pure.TicketId
import adr.spine.pure.Timestamp

class Sneak {
    fun page(now: Timestamp, ticket: TicketId) = EscalationEffect.PageOncall(now, ticket)
}
